database/seeders/DatabaseSeeder.php

```diff
@@ -23,5 +37,20 @@ public function run()
         $this->call(RoleSeeder::class);
         $this->call(UserSeeder::class);
         $this->call(IndustrySeeder::class);
+
```

Comment:
Should be useful to check the configuration env variable for if you want to install with demo data or not. If a production installation we need a clean database.

Reply:
if ( !app()->environment('production'))
{
        Account::factory(20)->create();
        Brand::factory(20)->create();
        Category::factory(20)->create();
        Industry::factory(20)->create();
        Lead::factory(20)->create();
        Customer::factory(20)->create();
        Contact::factory(20)->create();
        EmailTemplate::factory(20)->create();
        Email::factory(20)->create();
        Order::factory(20)->create();
        Product::factory(20)->create();
        Item::factory(20)->create();
        Supplier::factory(20)->create();
        Ticket::factory(20)->create();
}

Reply:
good idea!!

https://github.com/Roskus/hammercrm/pull/34/commits/eec15c2ac87fa0c716fb8075b83e54543ba7c2fe
In this commit I have separated the seeders according to the environment.

I hope it is what you were looking for.

I can get started with the tests if you want.